tests/examples/orphan-subprocess/playground/slow-startup/service

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+sleep 2
+touch readyfile
+exec sleep infinity
```

Comment:
there's no "orphan-subprocess" here.

Maybe you want `sleep 9999 &` as the first lien.